### PR TITLE
[DOCS] Update suite scaffold how-to

### PR DESCRIPTION
**Pull request by spbail**
Edited the scaffold how-to to be closer to my understanding of the workflow
- Moved the no-jupyter option further down, since I assume the default should be shown first in the how-to
- Added an example of an expectation string for included_expectations since it wasn't clear what to pass in there
- Rephrased the part that claims GE will choose expectations based on the column types, which isn't the case right now

Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [ENHANCEMENT], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- see above
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

Thank you for submitting!

### CODE REVIEW

**Comment by spbail** [comment]
Also @Aylr I assume minor doc changes don't need to go into the changelog?

**Comment by abegong** [comment]
👀

File: docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst [hunk 1]
```diff
@@ -22,26 +22,36 @@ Steps
...
 
         This command generates a disposable jupyter notebook in your ``great_expectations/uncommitted`` directory. The goal is to save you time by writing boilerplate code for you.
-
-        Because these notebooks can be generated at any time from the expectation suites (stored as JSON) you should **consider the notebook to be a disposable artifact**. You can delete it at any time.
+     
+        Because these notebooks can be generated at any time from the Expectation Suites (stored as JSON) you should **consider the notebook to be a disposable artifact**. You can delete it at any time.
 
     Once you choose a data asset, the CLI will open up a jupyter notebook.
+        
+        You can also run this command without opening the notebook by using the ` --no-jupyter` flag and then starting up jupyter separately:
```
Comment (abegong) [inline docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst]:
```suggestion
    You can also run this command without opening the notebook by using the `` --no-jupyter`` flag and then starting up jupyter separately:
```

Fixes indentation and RST markup.^^^

File: docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst [hunk 2]
```diff
@@ -22,26 +22,36 @@ Steps
...
         : npi.csv
-        To continue scaffolding this suite, run `jupyter notebook uncommitted/scaffold_npi_distributions.ipynb`
 
     .. important::
 
         This command generates a disposable jupyter notebook in your ``great_expectations/uncommitted`` directory. The goal is to save you time by writing boilerplate code for you.
-
-        Because these notebooks can be generated at any time from the expectation suites (stored as JSON) you should **consider the notebook to be a disposable artifact**. You can delete it at any time.
+     
+        Because these notebooks can be generated at any time from the Expectation Suites (stored as JSON) you should **consider the notebook to be a disposable artifact**. You can delete it at any time.
```
Comment (abegong) [inline docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst:35]: Nice catch

**Review by abegong — COMMENTED** [review]
This looks great. There's a minor formatting issue in line 39. Otherwise, it's perfect.

File: docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst [hunk 3]
```diff
@@ -63,21 +73,23 @@ Steps
...
+            "included_expectations": ['expect_column_values_to_not_be_null'],
             # "excluded_expectations": [],
         }
 
 3. **Generate Data Docs and review the results there**
 
     Run the next few code cells to see the scaffolded suite in Data Docs.
 
-    You can iterate on included and excluded columns and Expectations to get closer to the Suite you want.
+    Because the scaffolder is not very smart, you will want to edit this suite to tune the parameters and make any adjustments such as removing :ref:`Expectations` that don't make sense for your use case. You can iterate on included and excluded columns and Expectations to get closer to the Suite you want.
```
Comment (Aylr) [inline docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst]: Based on our pairing session yesterday @spbail I would probably recommend re-writing this to run the notebook as-is without adding `included_expectations` given that parameters not-so-great name.

**Review by Aylr — DISMISSED** [review]
Sam and I paired in a user testing session and we both came away with more understanding of use cases for the profiler. Based on this the doc may need to be adjusted to call less attention to the `included_expectations` configuration.

File: docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst [hunk 4]
```diff
@@ -63,21 +73,23 @@ Steps
...
+    
+    The next code cell shows the scaffold config, which contains the list of columns you included in the previous step, and the list of Expectations you want to generate (or not generate) for these columns. The list of included or excluded Expectations is a list of *strings* of the Expectation names, for example:
 
     .. code-block:: python
 
         scaffold_config = {
             "included_columns": included_columns,
             # "excluded_columns": [],
-            # "included_expectations": [],
+            "included_expectations": ['expect_column_values_to_not_be_null'],
```
Comment (jcampbell) [inline docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst]:
```suggestion
            # "included_expectations": [],
```

File: docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst (hunk 3 above)
Comment (jcampbell) [reply docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst]: I'm going to revert the specifically "included" expectation based on @Aylr 's recommendation for now, but please feel free to continue to suggest alternate edits.

**Review by jcampbell — APPROVED** [review]

**Pull request merged**